samples/eshop/infra/networking.bicep

```diff
@@ -4,10 +4,13 @@ import radius as rad
 @description('Radius application ID')
 param application string
 
+// GATEWAY RESOURCE NAME - Unique string generated from the application and gw
+param gatewayName string = 'gw${uniqueString(application)}'
```

Comment:
Can you double-check why this is needed? We should be generating unique URLs for each gateway that include the application name.

Reply:
Reverted this change and, yes, gateway name is gateway + appName @rynowak